### ISSUE TITLE
Can't browserify the package

### ISSUE THREAD

**Issue by themaric** [implydata/plywood-druid-requester#38]
Hi guys,

Can you please try to shed some light on the following issue?

```
$ browserify -r plywood-druid-requester
/home/my-project/node_modules/psl/index.js:1
import Punycode from 'punycode/punycode.js';
^
ParseError: 'import' and 'export' may appear only with 'sourceType: module'
```

Not sure whether it's directly related to this package, but found this in package-lock.json file:

```
"node_modules/@implydata/request/node_modules/tough-cookie": {
      "version": "4.1.3",
      "resolved": "https://registry.npmjs.org/tough-cookie/-/tough-cookie-4.1.3.tgz",
      "integrity": "sha512-aX/y5pVRkfRnfmuX+OdbSdXvPe6ieKX/G2s7e98f4poJHnqH3281gDPm/metm6E/WRamfx7WC4HUqkWHfQHprw==",
      "dependencies": {
        "psl": "^1.1.33",
        "punycode": "^2.1.1",
        "universalify": "^0.2.0",
        "url-parse": "^1.5.3"
      },
      "engines": {
        "node": ">=6"
      }
    },
```

So `punycode` (`tough-cookie`?) might need some love here by for example upgrading it to the latest release?

```
$ npm list plywood-druid-requester
my-project@ /home/my-project
└── plywood-druid-requester@3.0.2

$ npm list browserify
my-project@ /home/my-project
└── browserify@17.0.1
```

It'd be much appreciated if you could at least point me in the right direction so I can further investigate it.

Thanks!

Best regards,
Dusan

**Comment by themaric** [comment]
I'm just going to leave this here: https://github.com/lupomontero/psl/issues/326.

**Comment by lupomontero** [comment]
Hi @themaric :wave: 

This should have now been fixed in `psl@1.12.0`. Can you please check if this addresses the issue?

**Comment by themaric** [comment]
Hi,

Upgraded to v1.13.0 actually and the issue seems to be fixed.

Thanks for the follow up.

Cheers!
